### PR TITLE
Fixes #101 - Add schedule

### LINKED ISSUE #101: UXD - Add schedule styles (Desktop + Mobile)
'Show talk details' link switches to 'Close talk details' when any description(s) is expanded. 

Click on a talk title animates down to show description.

### PR DESCRIPTION
**Pull request by acacheung**
@Ctannerweb :eyes:

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -163,6 +163,101 @@
 </section>
 
 <span class="index__anchor" id="location"></span>
```
Comment (Ctannerweb) [inline views/index.erb]: change id to schedule

File: views/index.erb [hunk 2]
```diff
@@ -163,6 +163,101 @@
...
+          <time class="event__time" datetime="2015-06-16T16:50">04:50 pm</time>
+          <h3 class="event__name">Closing Keynote</h3>
+          <h4 class="event__speaker">Chris Tse</h4>
+        </a>
+      </li>
+    </ul>
+  </div>
+</section>
+
+<span class="anchor" id="location"></span>
```
Comment (Ctannerweb) [inline views/index.erb]: change class to `index__anchor`

File: views/index.erb [hunk 3]
```diff
@@ -163,6 +163,101 @@
...
+        <a class="event__link" href="/speakers#stefan-penner">
+          <time class="event__time" datetime="2015-06-16T15:30">03:30 pm</time>
+          <h3 class="event__name">The observer tip-jar</h3>
+          <h4 class="event__speaker">Stefan Penner</h4>
+        </a>
+      </li>
+      <li class="event">
+        <a class="event__link" href="/speakers#igor-terzic">
+          <time class="event__time" datetime="2015-06-16T16:10">04:10 pm</time>
+          <h3 class="event__name">Ember Data 1.0-it’s here</h3>
```
Comment (Ctannerweb) [inline views/index.erb]: Needs Smart quote

File: views/index.erb [hunk 4]
```diff
@@ -163,6 +163,101 @@
...
+      </li>
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T08:00">08:00 am</time>
+        <h3 class="event__name">Registration</h3>
+        <p class="event__desc">Sign in and get yourself prepared for the day’s events.</p>
+      </li>
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T09:00">09:00 am</time>
+        <h3 class="event__name">Opening</h3>
+        <p class="event__desc">A brief introduction to the day’s events.</p>
```
Comment (Ctannerweb) [inline views/index.erb]: Needs Smart quote

File: views/index.erb [hunk 5]
```diff
@@ -163,6 +163,101 @@
...
+    <ul class="events">
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T07:30">07:30 am</time>
+        <h3 class="event__name">Boat departure</h3>
+        <p class="event__desc">Get to the island from South Boston’s World Trade Center.</p>
+      </li>
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T08:00">08:00 am</time>
+        <h3 class="event__name">Registration</h3>
+        <p class="event__desc">Sign in and get yourself prepared for the day’s events.</p>
```
Comment (Ctannerweb) [inline views/index.erb]: Needs Smart quote

File: views/index.erb [hunk 6]
```diff
@@ -163,6 +163,101 @@
...
 
 <span class="index__anchor" id="location"></span>
+<section class="schedule">
+  <div class="l-schedule">
+    <h2 class="t-display">Schedule</h2>
+    <ul class="events">
+      <li class="event--general">
+        <time class="event__time" datetime="2015-06-16T07:30">07:30 am</time>
+        <h3 class="event__name">Boat departure</h3>
+        <p class="event__desc">Get to the island from South Boston’s World Trade Center.</p>
```
Comment (Ctannerweb) [inline views/index.erb]: Needs Smart quote

**Comment by Ctannerweb** [comment]
Anchor breakpoints need adjusted to new `max-height: 939px` and `min-height: 940px`

File: views/index.erb (hunk 3 above)
Comment (acacheung) [inline views/index.erb]: this should be how we do smart quotes
http://stackoverflow.com/questions/15776365/html-best-practices-should-i-use-rsquo-or-the-special-keyboard-shortcut
for reference we only need to escape & < and >
http://stackoverflow.com/questions/7381974/which-characters-need-to-be-escaped-on-html

**Pull request merged**